ENIGMAsystem/SHELL/Graphics_Systems/OpenGL-Common/textures.cpp

```diff
@@ -72,6 +72,9 @@ void graphics_delete_texture(int texid) {
 }
 
 unsigned char* graphics_copy_texture_pixels(int texture, unsigned* fullwidth, unsigned* fullheight) {
+  #ifndef glGetTexImage
+  return nullptr;
```

Comment:
Are you serious? FFS that means we either have to cache the texture in RAM or else read it back using the FBO (slow). We may as well just give up on deleting the texture data after upload because it's still just causing us so much pain.

This is covered by #1721 and #1727.